.circleci/config.yml

```diff
@@ -349,6 +349,98 @@ jobs:
           command: |
             docker push $DOCKER_REPO
 
+  # jobs for building chart          
+  helm-lint:
+    docker:
+      - image: lachlanevenson/k8s-helm
+    steps:
+      - checkout
+      - run:
+          name: Helm Lint
+          command: |
+            helm lint charts/mira
+
+  yaml-lint:
+    docker:
+      - image: sdesbure/yamllint
+    steps:
+      - checkout
+      - run:
+          name: YAML Lint
+          command: yamllint -c .yamllint.yml charts/mira/values.yaml
+
+  package:
+    working_directory: ~/charts
+    docker:
+      - image: lachlanevenson/k8s-helm
+    steps:
+      - checkout
+      - run:
+          name: Package
+          command: |
+            helm init --client-only
+            helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+            helm repo add qlik https://qliktech.jfrog.io/qliktech/qlikhelm --username $RT_USERNAME --password $RT_APIKEY
+
+            helm package charts/mira -d ~/charts
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - charts
+
+  push:
+    environment:
+      JFROG_CLI_LOG_LEVEL: "ERROR"
+      JFROG_CLI_OFFER_CONFIG: "false"
+    docker:
+      - image: circleci/python:2.7
+        auth:
+          username: $DOCKER_USERNAME
+          password: $DOCKER_PASSWORD
+    steps:
+      - attach_workspace:
+          at: ~/
+      - checkout
+      - run:
+          name: Push
+          command: |
+            #Check if the chart has been updated
+            result=$(python charts/check_git_py.py -c charts/mira)
```

Comment:
Can we check that the chart _version_ has been updated? I am thinking we want to make sure that the pushed chart is not the same version as the old one (someone fixing something small and forgets to update the version could easily happen)

Reply:
Yeah, that makes sense. Will push a change for that